hibernate-core/src/main/java/org/hibernate/type/descriptor/java/JdbcDateJavaTypeDescriptor.java

```diff
@@ -192,4 +215,16 @@ public int extractHashCode(Date value) {
 
 		throw unknownWrap( value.getClass() );
 	}
+
+	public static class DateMutabilityPlan extends MutableMutabilityPlan<Date> {
+		public static final DateMutabilityPlan INSTANCE = new DateMutabilityPlan();
+		@Override
+		public Date deepCopyNotNull(Date value) {
+			if ( value instanceof java.sql.Date ) {
+				return value;
```

Comment:
Should probably use this since it is a mutable type?

```suggestion
				return value.clone();
```